packages/flutter/lib/src/widgets/web_browser_detection.dart

```diff
@@ -0,0 +1,5 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+export '_web_browser_detection_io.dart' if (dart.library.js_util) '_web_browser_detection.dart';
```

Comment:
I didn't realize this was a pattern we did in the framework but looks like we do it in a couple of places already :+1:

Reply:
@Renzo-Olivares in order for this conditional import to work properly everywhere, the 2nd one needs to be renamed to `_web_browser_detection_web.dart` (the files basically have to suffixed by `_io.dart` and `_web.dart`).